### ISSUE TITLE
surface_format in "The Surface" breaking change for wpgu version 0.16.0

### ISSUE THREAD

**Issue by charliemonnone** [sotrh/learn-wgpu#473]
Hi, in "The Surface" section of the tutorial there is a bit of code that gets the surface format:
```
        let surface_format = surface_caps.formats.iter()
            .copied()
            .filter(|f| f.describe().srgb)
            .next()
            .unwrap_or(surface_caps.formats[0]);
```
The `wgpu` release for [0.16.0](https://github.com/gfx-rs/wgpu/releases/tag/v0.16.0) deprecates the describe function, so this line of code should now be:

`.filter(|f| f.is_srgb())`

**Comment by Antice** [comment]
There seems to have been a few changes to the tutorial since this issue was raised.
it now reads:
`        let surface_format = surface_caps.formats.iter()
            .copied()
            .find(|f| f.describe().srgb)            
            .unwrap_or(surface_caps.formats[0]);
`

Still using describe tho, So I guess this should be like this now?
`      let surface_format = surface_caps
            .formats
            .iter()
            .copied()
            .find(|f| f.is_srgb())
            .unwrap_or(surface_caps.formats[0]);
`

**Comment by adsick** [comment]
Not the matter, but that copied seems redundant to me, I guess we can clone data later if it is needed.

**Comment by sotrh** [comment]
Fixed in #493
